src/en/charms-actions.md

```diff
@@ -0,0 +1,317 @@
+# Juju Actions (DRAFT)
+
+Juju charms can describe actions that users can take on deployed services. 
+Actions are parametrized events that can be addressed and delivered to a 
+service or unit. Parameters for an action take the form of a YAML document,
+limited to things we can validate with JSON Schemas.
+
+The following juju commands are specified:
+
+    actions
+    do
+    wait
+    queue
+    kill
+
+In addition, the `status` command grows the ability to show status for 
+an action.
+
+
+
+
+## User experience
+
+A user can list the actions that a service or unit exports, as well as 
+the actions associated with interfaces to a service:
+
+    $ juju actions mysql
+    backup         benchmark      dump           ps             restart        
+    restore        start          stop           test
+    $ juju actions mysql/2
+    pause          resume         sync
+
+A user can take an action from the command-line passing an optional YAML
+file as parameters.
+
+    $ juju do mysql backup
```

Comment:
See above, there are examples for longer blocks of commands, I'll find an example.